server/meshmodel/schemas/core.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$id": "https://schema.meshery.io/core.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Reusable core schema elements",
+  "definitions": {
+    "inputString": {
+      "type": "string",
+      "pattern": "^[a-zA-Z_][a-zA-Z0-9_]*$"
```

Comment:
@sudo-NithishKarthik good. Is this pattern not applicable to other strings?

Reply:
I just made this pattern applicable to the ```name``` property as well.

Reply:
@harkiratsm every single field of type string should have a pattern assigned.